idasen/__init__.py

```diff
@@ -182,8 +182,17 @@ async def move_to_target(self, target: float):
                 f"{self.MIN_HEIGHT:.3f}"
             )
 
+        previous_height = await self.get_height()
+        will_move_up = target > previous_height
         while True:
             height = await self.get_height()
+            if (height < previous_height and will_move_up) or (
+                height > previous_height and not will_move_up
+            ):
+                self._logger.info(
```

Comment:
```suggestion
                self._logger.warning(
```
I think this should be a higher level than `info` since it is unexpected.